qsp_transfer.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import traceback
+import json
+import boto3
+import base64
+import time
+import re
+import logging
+
+from ftplib import FTP
+from datetime import datetime, timedelta
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def get_ftp_credentials():
+    client = boto3.client('secretsmanager')
+    get_secret_value_response = client.get_secret_value(SecretId=os.environ['SECRET_NAME'])
```

Comment:
We should probably do some exception handling here

Reply:
Exceptions were originally being caught and re-raised here in a `try/except` block but this was [simplified](https://github.com/companieshouse/ois-tuxedo-qsp-transfer/pull/1/commits/0c37f93242d0265028f039ca02eabef4c3a62da8) in favour of allowing exceptions to bubble up the call stack to be dealt with in the 'global' `try/except` block present in the handler function.

Since we don't need to respond to any particular exception with specific logic, all exceptions will be caught and logged in the handler including structured data for the error type, error message and call stack. This also removes the need for additional `try/except` blocks here and in other places that may throw an exception.

We could reintroduce the `try/except` block here, and other places that generate exceptions, but at the cost of some readability and some duplication.